docs/content/users/basics/faq.md

```diff
@@ -109,6 +109,12 @@ Start by completely turning NFS off for your projects with `ddev config --nfs-mo
 
 You’ll want to update DDEV using the same method you chose to install it. Since upgrading is basically the same as installing, you can follow [DDEV Installation](../install/ddev-installation.md) to upgrade.
 
+DDEV can help you figure out how you installed and how you can upgrade:
```

Comment:
Link to the command reference for two reasons:

1. Reiterates how DDEV works (with commands).
2. Offers a more detailed example than the one otherwise isolated here.

```suggestion
You can use the [`self-upgrade`](../basics/commands.md#self-upgrade) command for getting instructions tailored to your installation.
```